src/css/components/Avatar.scss

```diff
@@ -24,4 +24,9 @@ $sizes: (
       height: $value;
     }
   }
+
+  &--fill {
```

Comment:
Don't think we actually need this modifier. In the header, the image can be centered vertically by setting it to display block (will remove the extra space below it). I also think we probably want this to have a fixed width so it doesn't scale in size with the screen.

```
.Nav-headerIcon {
  width: scaleGrid(3);
  display: block;
}

<div class="Nav-header">
  <img class="Nav-headerIcon"/>
</div>
```

Reply:
This modifier on the avatar component will also solve an issue that we're seeing in org, but I've added the CSS that you've recommended and got it behaving as expected, as well.